src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -1986,6 +1986,45 @@ protected override void AfterMembersCompletedChecks(BindingDiagnosticBag diagnos
                     var syntax = (ExtensionBlockDeclarationSyntax)this.GetNonNullSyntaxNode();
                     diagnostics.Add(ErrorCode.ERR_BadExtensionContainingType, syntax.Keyword);
                 }
+
+                if (TryGetOrCreateExtensionMarker() is { Parameters: [var extensionParameter] })
+                {
+                    checkUnderspecifiedGenericExtension(extensionParameter, TypeParameters, diagnostics);
+                }
+            }
+
+            return;
+
+            void checkUnderspecifiedGenericExtension(ParameterSymbol parameter, ImmutableArray<TypeParameterSymbol> typeParameters, BindingDiagnosticBag diagnostics)
+            {
+                if (GetMembers().All((m, a) => m is MethodSymbol { MethodKind: MethodKind.Ordinary }, arg: (object?)null))
```

Comment:
>All [](http://example.com/codeflow?start=33&length=3)

Is there an overload that doesn't need unused additional argument? Perhaps we have ```Any``` instead?